### ISSUE TITLE
Problem about controling Jaguar2015XE via keyboard in CubeTown in Unity3D.

### ISSUE THREAD

**Issue by wobushigaoying** [lgsvl/simulator#1353]
Hello, I followed the instruction of "building XXX from source", and that content did work correctly.

I then cloned CubeTown, Jaguar2015XE, and KeyboardControlSensor from Github. I put all these assets in the right"External" folder.

I dragged the CubeTown prefab into the Hierarchy tab in Unity3D, as well as the Jaguar2015XE prefab and SimulatorManager prefab. I also dragged the KeyboardControlSensor prefab into Jaguar2015XE as a sub-component in the Hierarchy tab. However, the simulation did not work after I click RUN. Here are the errors:

![image](https://user-images.githubusercontent.com/18672103/114124203-7f640380-9926-11eb-9754-c6fab02d6450.png)
![image](https://user-images.githubusercontent.com/18672103/114124216-83902100-9926-11eb-960a-d2587a7db782.png)
![image](https://user-images.githubusercontent.com/18672103/114124221-87bc3e80-9926-11eb-98d0-476165576bd3.png)
![image](https://user-images.githubusercontent.com/18672103/114124230-8db21f80-9926-11eb-8503-696dab4496f6.png)
![image](https://user-images.githubusercontent.com/18672103/114124234-91de3d00-9926-11eb-87a3-491d7c6babdc.png)
![image](https://user-images.githubusercontent.com/18672103/114124241-94d92d80-9926-11eb-8cb4-f0d9e89ffd4a.png)
![image](https://user-images.githubusercontent.com/18672103/114124250-99054b00-9926-11eb-904d-d63b06e8ce52.png)
![image](https://user-images.githubusercontent.com/18672103/114124256-9c98d200-9926-11eb-8e10-edd519ffc821.png)
![image](https://user-images.githubusercontent.com/18672103/114124259-9f93c280-9926-11eb-83f7-d203d478f251.png)

All these errors seem to refer to only one type of error, that is "NullReferenceException". I checked all the relevant scripts and prefabs, and I found all the references were actually existed and cited correctly. However, the simulation did not run as I expected. 

Would you mind helping me debugging this issue? I would be very grateful for your generosity.

By the way, I am using Unity3D version 2019.4.18f1, and LGSVL 2021.01. All the assets I cloned from GitHub were the latest released version.

Many thanks.

**Comment by EricBoiseLGSVL** [comment]
Hello, thanks for trying the latest release of svl simulator.  Simulator work flow has changed quite a bit.  To start simulations in the editor:

1) Be sure to sign up at svlsimulator.com first.  This makes it easy to link the editor to the webUI.
2) Open Loader.scene in Assets/Scenes/ and run the editor
3) You should see a link to cloud button, click it
4) This will open the web browser and ask to login. Do so.  If you are already logged in it will ask to link the machine to the cluster.  Add a name for the machine, I always choose Local.
5) If you haven't logged in before opening the webUI, I recommend stopping Editor after logging in and then playing again to be at cluster naming step.
6) Create simulation and add vehicles, maps, sensors etc and start sim.

Please see documentation [here](https://www.svlsimulator.com/docs/getting-started/getting-started/). The work flow is the same but for editor be sure to start in Loader scene.  All assets are loaded from webUI and attached to vehicle, no need to do it in scene.

**Comment by wobushigaoying** [comment]
Hello, thanks for your remainder. I will try your tips soon. 

Hope the simulation will correctly run in Unity3D.

BTW, I noticed "editor" in your reply and some other issues. Does the "editor" you mentioned indicate "Unity3D"?  

Is it convenient to send me projects related to the videos "https://www.youtube.com/watch?v=R_Z65kimCII" and "https://www.youtube.com/watch?v=S0w2ahhEPbE"?

Again, many thanks.

**Comment by EricBoiseLGSVL** [comment]
Yes, Unity Editor is usually how it is referenced. We also call it developer mode in the documentation.  It will run in editor no worries, I use it every day :)  Hope the videos are helpful. Let us know if you get stuck.
